docker/blackduck-alert-db/Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM postgres:12.2-alpine
 
+ARG VERSION
+
+LABEL com.blackducksoftware.integration.alert.vendor="Black Duck Software, Inc." \
+      com.blackducksoftware.integration.alert.version="$VERSION"
+
 ENV DOCKER_ENTRYPPOINT_INITD_DIR /docker-entrypoint-initdb.d
 
 RUN mkdir -p -m 774 $DOCKER_ENTRYPPOINT_INITD_DIR
 
-COPY alertdb-healthcheck.sh /usr/local/bin/alertdb-healthcheck.sh
+COPY --chmod=444 alertdb-healthcheck.sh /usr/local/bin/alertdb-healthcheck.sh
```

Comment:
Do we no longer require write/execute permissions for the alertdb-healthcheck?

Reply:
And this is why I want reviews! Thank you @ChomickiM